#request
float64 x
float64 y
float64 z
geometry_msgs/Pose pose
---
#response
bool status